### DISCUSSION TITLE
Sample Config Explaination

### DISCUSSION THREAD

**Discussion by ProjectInitiative** [seaweedfs/seaweedfs-operator#54] [Q&A]
I have read through some of the issues/discussion and know some documentation changes are in the works. I wanted to inquire about the [volumeServerDiskCount](https://github.com/seaweedfs/seaweedfs-operator/blob/cbb981aad9ff37bb975a8461956f538407aebf75/config/samples/seaweed_v1_seaweed.yaml#L9) setting in the provided sample and see if this was like a physical disk count or something different. I seen the enhancement in [issue 40](https://github.com/seaweedfs/seaweedfs-operator/issues/40) for more granular control over the volume server deployment and I am also run an onprem server stack with no smart tech over all of my disks (no RAID or ZFS). I was wondering if anything is currently in the works for handling node deployments to with host paths (say to individual disks) or if this is already supported, getting a nudge in the right direction. I am planning on eventually adding a HDD and SSD tier with host paths and trying to evaluate if foregoing the operator and managing the k8s configs manually is the way to go at the present time.

Thanks!

**Comment by chrislusf** [comment]
volumeServerDiskCount is like a physical disk count. https://github.com/seaweedfs/seaweedfs-operator/blob/1053ae7d62c06b7a155aad10faf3f1490e375eba/controllers/controller_volume_statefulset.go#L45

there are no assumption for special type of RAID or file system.

**Reply by ProjectInitiative** [reply]
Gotcha, so if one wanted to specify multiple "disks" with this operator, would multiple volume specs provide this functionality?
